packages/atomic/src/components/atomic-search-interface/atomic-aria-live.tsx

```diff
@@ -1,40 +1,80 @@
-import {Component, h, Host, Listen, State} from '@stencil/core';
+import {Component, h, Host, State, Element, Method, Prop} from '@stencil/core';
 
-export interface UpdateLiveRegionEventArgs {
-  region: string;
-  message: string;
+export interface FindAriaLiveEventArgs {
+  element?: HTMLAtomicAriaLiveElement;
 }
 
 /**
  * The `atomic-aria-live` component notifies screen readers of changes in the search interface.
- * @internal
+ *
+ * It is recommended not to dynamically add/remove this component from the page and try to have it exist on the initial render of the page.
  */
 @Component({
   tag: 'atomic-aria-live',
   shadow: false,
 })
 export class AtomicAriaLive {
-  @State() public regions: Record<string, string> = {};
+  @Element() private host!: HTMLAtomicAriaLiveElement;
+  @State() private message = '';
 
-  @Listen('atomic/accessibility/updateLiveRegion', {target: 'document'})
-  protected updateMessage({
-    detail: {message, region},
-  }: CustomEvent<UpdateLiveRegionEventArgs>) {
-    this.regions = {...this.regions, [region]: message};
+  /**
+   * Only the `atomic-aria-live` element with the greatest priority will be used to announce changes in the search interface.
```

Comment:
The priority system might be a bit overkill, at least at first.

Reply:
it's definitely overkill, I agree, which is why I made it internal. I added it pretty much exclusively to differentiate between an `atomic-aria-live` inside the all search interfaces and one inside. Should I use a different mechanism?

Reply:
If it's not needed or overkill, I propose to remove it.

We should not add props and option because we think we might possibly need it in in the future. 
Only when it's necessary.